src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ArrayTrailingCommaCheck.java

```diff
@@ -162,8 +162,7 @@ public void visitToken(DetailAST arrayInit) {
         final DetailAST rcurly = arrayInit.findFirstToken(TokenTypes.RCURLY);
         final DetailAST previousSibling = rcurly.getPreviousSibling();
 
-        if (arrayInit.getLineNo() != rcurly.getLineNo()
```

Comment:
When `arrayInit.getLineNo() == rcurly.getLineNo()` (not of code above) is `true`, `arrayInit.getLineNo() != previousSibling.getLineNo()` always fail as previous sibling is always on the same line since the 2 curlies are on the same line.